### PR TITLE
spread, data/selinux: add CentOS 8, update policy

### PR DESCRIPTION
**Pull request by bboozzoo**
Add CentOS 8 to systems list.

### CODE REVIEW

**Comment by sergiocazzolato** [comment]
@bboozzoo hey, please try again because I already update the image centos-8-64

**Comment by sergiocazzolato** [comment]
still 3 tests failing: 
2020-02-04 15:32:46 Failed tasks: 3
    - google:centos-8-64:tests/main/interfaces-packagekit-control
    - google:centos-8-64:tests/upgrade/basic
    - google:centos-8-64:tests/upgrade/selinux-relabel

An error which appears twice is:
distro_install_package snap-confine snapd
quiet: dnf -y --refresh install snap-confine snapd
quiet: exit status 1. Output follows:
....
No match for argument: snap-confine
No match for argument: snapd
Error: Unable to find a match: snap-confine snapd
quiet: end of output.

Also is failing to get the gpg key for repo 'google-compute-engine'

**Comment by bboozzoo** [comment]
cc @Conan-Kudo added a tiny update for the policy in case you want to take a look

File: spread.yaml [hunk 1]
```diff
@@ -596,12 +604,12 @@ prepare: |
...
                 apt-get install -y xdelta3 curl eatmydata >& "$tf" || ( cat "$tf"; exit 1 )
                 ;;
-            fedora-*)
-                dnf install --refresh -y xdelta curl &> "$tf" || (cat "$tf"; exit 1)
-                ;;
-            amazon-*|centos-*)
+            amazon-*|centos-7-*)
                 yum install -y xdelta curl &> "$tf" || (cat "$tf"; exit 1)
                 ;;
+            fedora-*|centos-*)
```
Comment (Conan-Kudo) [inline spread.yaml]: Won't this run again on `centos-7` machines? This will fail unless you preinstall `dnf` on the CentOS 7 machines...

Comment (bboozzoo) [reply spread.yaml]: No, we use `<sys>-<version>-<arch>` naming pattern, so CentOS systems are named `centos-7-64` (well, there's no 32 variant TBH), so it hits the case clause right above. Anything below 7 isn't really supported, so there's no spread systems defined for that.

**Pull request merged**